Header header
geometry_msgs/Point center
geometry_msgs/Point initial
float64 rotation_rate
float64 outward_angle